src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &--1 :first-child,
+  &--2 :nth-child(-n + 2),
+  &--3 :nth-child(-n + 3),
+  &--4 :nth-child(-n + 4),
+  &--5 :nth-child(-n + 5) {
+    background-image: url(/src/images/star-active.svg);
```

Comment:
It's better to use relative paths without leading slashes for images to avoid potential issues with different base paths on various environments. Use `background-image: url(../images/star-active.svg);` instead.